velox/functions/lib/window/tests/WindowTestBase.cpp

```diff
@@ -100,7 +100,7 @@ RowVectorPtr WindowTestBase::makeRandomInputVector(vector_size_t size) {
       {makeRandomInputVector(BIGINT(), size, 0.2),
        makeRandomInputVector(VARCHAR(), size, 0.3),
        makeFlatVector<int64_t>(size, genRandomFrameValue),
-       makeFlatVector<int64_t>(size, genRandomFrameValue)});
+       makeFlatVector<int32_t>(size, genRandomFrameValue)});
```

Comment:
Is this a fix for unrelated issue? Can it be extracted into a separate PR?

Reply:
This change was made to test the columnar int32_t offset in spark tests (`nth_value(c0, c3)`), since the check for constant integer type offset in spark is removed in this PR. Please let me know if it should be reverted.